Sources/Plasma/FeatureLib/pfPython/pyVaultNode.cpp

```diff
@@ -119,7 +119,7 @@ void pyVaultNode::pyVaultNodeOperationCallback::VaultOperationComplete( uint32_t
                 pyObjectRef tup = PyTuple_New(2);
                 PyTuple_SET_ITEM(tup.Get(), 0, pyVaultNode::New(fNode));
                 PyTuple_SET_ITEM(tup.Get(), 1, fPyNodeRef.Release());
-                plPython::CallObject(func, context, std::move(tup), resultCode);
+                plPython::CallObject(func, context, std::move(tup), static_cast<int32_t>(result));
```

Comment:
I tried, yes. :P I think I ran into problems with references when defining the overload outside of plPythonConvert.h - perhaps it will work better if I add it directly in the header.

Reply:
Nevermind, I just tried again and it works either way now... 🤷‍♀️

Reply:
Those overloads can be very fragile for reasons that I don't understand completely 😞